docs/setup/setup.md

```diff
@@ -190,6 +190,19 @@ We have provided a sample node.json to add a node in kubernetes. Please make sur
 
     **Note:** Please run edge using the users who have root permission.
 
++ Run edge with systemd
```

Comment:
nit: Please add a blank line after this line so that the content starts from next line in the doc